scripts/webui.py

```diff
@@ -940,11 +940,10 @@ def process_images(
             if opt.optimized:
                 modelFS.to(device)
 
+            for i in range(batch_size):
```

Comment:
Maybe    range(len(samples_ddim))     instead of batch_size. With prompt matrix it is possible to have a total number of generations which is not a multiple of batch size, which will crash in samples_ddim[i]